core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
Thanks @ijuma. I'm just thinking about a perhaps rare or unlikely scenario in which, the user has currently set their partition assignment strategy to `roundrobin`. Later after the major release (when default is changed) they decide to go back to `range` assuming that is still the default. This way, they don't see any warning (as the warning will supposedly be removed when the default is actually changed) and will keep using the `roundrobin` assignment even though they think they're using `range`.

Very unlikely scenario, but just wanted to make sure it is still ok to go with your suggestion. Thanks.